bridges/evm/contracts/Bridge.sol

```diff
@@ -99,6 +99,32 @@ contract Bridge is IBridge {
     bytes32 dataHash;
   }
 
+  /// Decode the encoded result and receives back the decoded data which is the data and its context.
+  /// @param _data The encoded of result and its context.
+  function decodeResult(bytes memory _data) public pure returns (VerifyOracleDataResult memory) {
+    require(_data.length > 40, "INPUT_MUST_BE_LONGER_THAN_40_BYTES");
+
+    VerifyOracleDataResult memory result;
+    assembly {
+      mstore(add(result, 0x20), and(mload(add(_data, add(0x08, 0x00))), 0xffffffffffffffff))
+      mstore(add(result, 0x40), and(mload(add(_data, add(0x08, 0x08))), 0xffffffffffffffff))
+      mstore(add(result, 0x60), and(mload(add(_data, add(0x08, 0x10))), 0xffffffffffffffff))
+      mstore(add(result, 0x80), and(mload(add(_data, add(0x08, 0x18))), 0xffffffffffffffff))
+      mstore(add(result, 0xa0), and(mload(add(_data, add(0x08, 0x20))), 0xffffffffffffffff))
+    }
+
+    bytes memory data = new bytes(_data.length - 40);
+    uint256 l = ((data.length - 1) / 32) + 1;
+    for (uint256 i = 0; i < l; i++) {
+      assembly {
+        mstore(add(data,add(0x20, mul(i,0x20))), mload(add(_data, add(0x48, mul(i,0x20)))))
+      }
+    }
+    result.data = data;
+
+    return  result;
```

Comment:
space?

Reply:
Right now we are using auto solidity format